api/exceptions.py

```diff
@@ -10,6 +10,17 @@ class ConflictError(APIException):
     default_code = "conflict_error"
 
 
+class DomainAddressConflictError(ConflictError):
+    default_detail = "This domain address already exists."
```

Comment:
(question, non-blocking) I forget what our rules are around adding translation strings to exceptions being raised. Wondering if you have experience and if this needs to be added to translation strings? My initial thought is that if this is user-facing we should add string for translation.

Reply:
The rules are the ones we make! I think we worked to make exceptions translated for those that would get to the Firefox integration. Since we don't have a way to create domain addresses, this one is in the grey area. Still, I think it would be good to get into the habit of making these translated. That would mean basing it on `RelayAPIException`, not `ConflictError`.